ex-pod.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: standalone-test-pod
+  #namespace: jose-matsuda
+  namespace: mathis-marcotte 
```

Comment:
This file wont get commited, right?

Reply:
I mainly kept it for easy testing of the webhook without having to start up a pod, I would like to keep it tbh. Maybe I will just have it documented somewhere